mod/cli/pkg/commands/jwt/jwt.go

```diff
@@ -117,16 +118,14 @@ func getFilePath(cmd *cobra.Command, path string) (string, error) {
 	// If no path is specified, try to get the cosmos client context and use
 	// the configured home directory to write the secret to the default file
 	// name.
-	if specifiedFilePath == "" {
```

Comment:
i don't think this is logically equivalent

Reply:
is it not?

Reply:
we already early return if `specifiedFilePath` is not empty [here](https://github.com/berachain/beacon-kit/blob/056a0a5f53a6898672ad44048dd997693a8a66e6/mod/cli/pkg/commands/jwt/jwt.go#L114) so no need to check again.